### PR TITLE
added responsibilities page

### PR DESCRIPTION
**Pull request by wslack**

### CODE REVIEW

**Comment by wslack** [comment]
(anyone can review, assigning Brandon for now)

File: pages/using-federalist/customer-responsibilities.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+title: Customer Responsibilities
+parent: Using Federalist
+---
+
+## Your Responsiibilities when using Federalist
```
Comment (brandocalrissian) [inline pages/using-federalist/customer-responsibilities.md]: Responsiibilities --> Responsibilities (drop the extra first `i`)

File: pages/using-federalist/customer-responsibilities.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+Using Federalist places certain responsibilities on you as the governmnet user of Federalist. GSA has many internal sites running on Federalist, but GSA cannot guarantee that Federalist is compatable with your agency's specific policies. 
+
+Federalist retains other responsibilities, clarified below. The intent of these policies is to empower you to use Federalist to its full potential with awareness of your responsibilities when using advanced features.
+
+#### You must use a GitHub account to log onto Federalist.
+
+Federalist is a service that leverages GitHub repositories for publishing. Federalist elimates redundancy by allowing GitHub users with editing access to a repository to configure the site on Federalist.
+
+GitHub is used across the government (see [this dashboard](http://gsa.github.io/github-federal-stats/) from our partners in GSA's Office of Government-wide Policy), and a majority of cabinent agencies have Github presence. However, GSA does not endorse Github and there are other ways to launch static sites if your agency is not prepared to use GitHub. At this time, no propsective Federalist customer has been deterred by integration with GitHub.
```
Comment (brandocalrissian) [inline pages/using-federalist/customer-responsibilities.md]: can we link to the https version instead (https://gsa.github.io/github-federal-stats/)

**Comment by brandocalrissian** [comment]
Looks good other than the two minor comments above.

**Comment by wslack** [comment]
Merging per @brandocalrissian.

**Pull request merged**